diff --git a/node_modules/@rn-primitives/dialog/dist/dialog.d.ts b/node_modules/@rn-primitives/dialog/dist/dialog.d.ts
index 3d381e9..46e36a6 100644
--- a/node_modules/@rn-primitives/dialog/dist/dialog.d.ts
+++ b/node_modules/@rn-primitives/dialog/dist/dialog.d.ts
@@ -3,6 +3,7 @@ import * as react_native from 'react-native';
 import { View, Text } from 'react-native';
 import * as React from 'react';
 import { PortalProps, RootContext } from './index.js';
+import { PortalHost } from "@rn-primitives/portal";
 import './dialog';

 declare const Root: React.ForwardRefExoticComponent<react_native.ViewProps & {
@@ -55,4 +56,4 @@ declare const Description: React.ForwardRefExoticComponent<react_native.TextProp
     asChild?: boolean;
 } & React.RefAttributes<Text>>;

-export { Close, Content, Description, Overlay, Portal, Root, Title, Trigger, useRootContext };
+export { Close, Content, Description, Overlay, Portal, PortalHost, Root, Title, Trigger, useRootContext };
diff --git a/node_modules/@rn-primitives/dialog/dist/dialog.js b/node_modules/@rn-primitives/dialog/dist/dialog.js
index 4851de2..6a81722 100644
--- a/node_modules/@rn-primitives/dialog/dist/dialog.js
+++ b/node_modules/@rn-primitives/dialog/dist/dialog.js
@@ -36,6 +36,7 @@ __export(dialog_exports, {
   Description: () => Description,
   Overlay: () => Overlay,
   Portal: () => Portal,
+  PortalHost: () => import_portal.PortalHost,
   Root: () => Root,
   Title: () => Title,
   Trigger: () => Trigger,
@@ -198,6 +199,7 @@ function onStartShouldSetResponder() {
   Description,
   Overlay,
   Portal,
+  PortalHost: import_portal.PortalHost,
   Root,
   Title,
   Trigger,
diff --git a/node_modules/@rn-primitives/dialog/dist/dialog.mjs b/node_modules/@rn-primitives/dialog/dist/dialog.mjs
index 9108d09..866ab9b 100644
--- a/node_modules/@rn-primitives/dialog/dist/dialog.mjs
+++ b/node_modules/@rn-primitives/dialog/dist/dialog.mjs
@@ -2,7 +2,7 @@

 // src/dialog.tsx
 import { useControllableState } from "@rn-primitives/hooks";
-import { Portal as RNPPortal } from "@rn-primitives/portal";
+import { Portal as RNPPortal, PortalHost } from "@rn-primitives/portal";
 import * as Slot from "@rn-primitives/slot";
 import * as React from "react";
 import { BackHandler, Pressable as Pressable2, Text, View as View2 } from "react-native";
@@ -156,6 +156,7 @@ export {
   Description,
   Overlay,
   Portal,
+  PortalHost,
   Root,
   Title,
   Trigger,
