diff --git a/node_modules/@rn-primitives/select/dist/select.d.mts b/node_modules/@rn-primitives/select/dist/select.d.mts
index 3292308..d1801a6 100644
--- a/node_modules/@rn-primitives/select/dist/select.d.mts
+++ b/node_modules/@rn-primitives/select/dist/select.d.mts
@@ -3,6 +3,7 @@ import { ItemTextProps, PortalProps, Option, ScrollDownButtonProps, ScrollUpButt
 import * as react_native from 'react-native';
 import { View, Text, LayoutRectangle } from 'react-native';
 import { LayoutPosition } from '@rn-primitives/hooks';
+import { PortalHost } from "@rn-primitives/portal";
 import * as React from 'react';
 import './select';

@@ -92,4 +93,4 @@ declare const ScrollUpButton: ({ children }: ScrollUpButtonProps) => React.JSX.E
 declare const ScrollDownButton: ({ children }: ScrollDownButtonProps) => React.JSX.Element;
 declare const Viewport: ({ children }: ViewportProps) => React.JSX.Element;

-export { Content, Group, Item, ItemIndicator, ItemText, Label, Overlay, Portal, Root, ScrollDownButton, ScrollUpButton, Separator, Trigger, Value, Viewport, useItemContext, useRootContext };
+export { Content, Group, Item, ItemIndicator, ItemText, Label, Overlay, Portal, PortalHost, Root, ScrollDownButton, ScrollUpButton, Separator, Trigger, Value, Viewport, useItemContext, useRootContext };
diff --git a/node_modules/@rn-primitives/select/dist/select.d.ts b/node_modules/@rn-primitives/select/dist/select.d.ts
index 4e857dd..48d5bf6 100644
--- a/node_modules/@rn-primitives/select/dist/select.d.ts
+++ b/node_modules/@rn-primitives/select/dist/select.d.ts
@@ -3,6 +3,7 @@ import { ItemTextProps, PortalProps, Option, ScrollDownButtonProps, ScrollUpButt
 import * as react_native from 'react-native';
 import { View, Text, LayoutRectangle } from 'react-native';
 import { LayoutPosition } from '@rn-primitives/hooks';
+import { PortalHost } from "@rn-primitives/portal";
 import * as React from 'react';
 import './select';

@@ -92,4 +93,4 @@ declare const ScrollUpButton: ({ children }: ScrollUpButtonProps) => React.JSX.E
 declare const ScrollDownButton: ({ children }: ScrollDownButtonProps) => React.JSX.Element;
 declare const Viewport: ({ children }: ViewportProps) => React.JSX.Element;

-export { Content, Group, Item, ItemIndicator, ItemText, Label, Overlay, Portal, Root, ScrollDownButton, ScrollUpButton, Separator, Trigger, Value, Viewport, useItemContext, useRootContext };
+export { Content, Group, Item, ItemIndicator, ItemText, Label, Overlay, Portal, PortalHost, Root, ScrollDownButton, ScrollUpButton, Separator, Trigger, Value, Viewport, useItemContext, useRootContext };
diff --git a/node_modules/@rn-primitives/select/dist/select.js b/node_modules/@rn-primitives/select/dist/select.js
index 2c68c40..24a30b8 100644
--- a/node_modules/@rn-primitives/select/dist/select.js
+++ b/node_modules/@rn-primitives/select/dist/select.js
@@ -39,6 +39,7 @@ __export(select_exports, {
   Label: () => Label,
   Overlay: () => Overlay,
   Portal: () => Portal,
+  PortalHost: () => import_portal.PortalHost,
   Root: () => Root,
   ScrollDownButton: () => ScrollDownButton,
   ScrollUpButton: () => ScrollUpButton,
@@ -368,6 +369,7 @@ function onStartShouldSetResponder() {
   Label,
   Overlay,
   Portal,
+  PortalHost: import_portal.PortalHost,
   Root,
   ScrollDownButton,
   ScrollUpButton,
diff --git a/node_modules/@rn-primitives/select/dist/select.mjs b/node_modules/@rn-primitives/select/dist/select.mjs
index 5756165..de202b4 100644
--- a/node_modules/@rn-primitives/select/dist/select.mjs
+++ b/node_modules/@rn-primitives/select/dist/select.mjs
@@ -6,7 +6,7 @@ import {
   useControllableState,
   useRelativePosition
 } from "@rn-primitives/hooks";
-import { Portal as RNPPortal } from "@rn-primitives/portal";
+import { Portal as RNPPortal, PortalHost } from "@rn-primitives/portal";
 import * as Slot from "@rn-primitives/slot";
 import * as React from "react";
 import {
@@ -327,6 +327,7 @@ export {
   Label,
   Overlay,
   Portal,
+  PortalHost,
   Root,
   ScrollDownButton,
   ScrollUpButton,
